library/core/src/main/java/com/google/android/exoplayer2/text/ttml/TtmlDecoder.java

```diff
@@ -250,11 +254,29 @@ private CellResolution parseCellResolution(XmlPullParser xmlParser, CellResoluti
         if (ttmlRegion != null) {
           globalRegions.put(ttmlRegion.id, ttmlRegion);
         }
+      } else if(XmlPullParserUtil.isStartTag(xmlParser, TtmlNode.TAG_METADATA)){
+        parseMetaData(xmlParser, imageMap);
       }
+
     } while (!XmlPullParserUtil.isEndTag(xmlParser, TtmlNode.TAG_HEAD));
     return globalStyles;
   }
 
+  public void parseMetaData(XmlPullParser xmlParser, Map<String, String> imageMap) throws IOException, XmlPullParserException {
```

Comment:
nit - We consider metadata to be one word, so should be `Metadata` rather than `MetaData`.

Reply:
This method should be private.